_docs/compliance/logging-requirements.md

```diff
@@ -0,0 +1,60 @@
+---
+parent: compliance
+layout: docs
+sidenav: true
+title: Logging Requirements
+---
+
+How our system meets logging requirements specified by [Executive Order M-21-31](https://www.whitehouse.gov/wp-content/uploads/2021/08/M-21-31-Improving-the-Federal-Governments-Investigative-and-Remediation-Capabilities-Related-to-Cybersecurity-Incidents.pdf).
```

Comment:
Rephrase to say that we are enabling our customers to comply with M-21-31

Reply:
Add: If you have questions about the implementation, reach out to support@cloud.gov